tests/datasource/fluent/integration/test_connections.py

```diff
@@ -23,10 +23,6 @@ class TestSnowflake:
     @pytest.mark.parametrize(
         "connection_string",
         [
-            param(
-                "snowflake://ci:${SNOWFLAKE_CI_USER_PASSWORD}@oca29081.us-east-1/ci/public?warehouse=ci",
-                id="missing role",
-            ),
```

Comment:
@JennyTee tagging you because these test cases are going away.

Users will hit config errors if they are missing `role` or `warehouse` now before even trying to test the connection.